apache-solr/jinja/templates/cloud-init-template.yaml

```diff
@@ -0,0 +1,61 @@
+# cloud-init-template.yaml
+
+# Create Grafana Agent configuration
+write_files:
+  - owner: root:root
+    path: /etc/grafana-agent.yaml
+    content: |
+      integrations:
+        node_exporter:
+          enabled: true
+          relabel_configs:
+          - replacement: hostname
+            target_label: instance
+        prometheus_remote_write:
+        - basic_auth:
+            password: "{{ prom_pass }}"
+            username: "{{ prom_user }}"
```

Comment:
This should be conditional. If the prom_user and prom_pass are unset, then it implies that no authentication is required.

An example from the MySQL [jinja template.](https://github.com/grafana/integration-sample-apps/blob/c0916d21dae6b9d130af99e00226e0954609b450/mysql/jinja/templates/cloud-init-template.yaml#L34-L39)

Reply:
updated